segway/run.py

```diff
@@ -4201,10 +4209,62 @@ def parse_options(argv):
     return task_spec, options, archives, traindir, annotatedir
 
 
+def check_gmtk_version():
+    """ Checks if the supported minimum GMTK version is installed """
+    # Typical expected output from "gmtkPrint -version":
+    # gmtkPrint (GMTK) 1.4.3
+    # Mercurial id: 8995e40101d2 tip
+    # checkin date: Fri Oct 30 10:51:44 2015 -0700
+
+    # Older versions:
+    # GMTK 1.4.0 (Mercurial id: bdf2718cc6ce tip checkin date: Thu Jun 25 12:31:56 2015 -0700)
+
+    # Try to open gmtkPrint to get the version
+    try:
+        # blocks until finished
+        output_string = subprocess.check_output(["gmtkPrint", "-version"])
```

Comment:
Can you use `optbuild` for this instead? I don't think we use raw `subprocess` anywhere else in Segway and would be good to be consistent.